lib/receiver.js

```diff
@@ -586,7 +584,7 @@ module.exports = Receiver;
 /**
  * Builds an error object.
  *
- * @param {(Error|RangeError)} ErrorCtor The error constructor
+ * @param {(typeof Error|typeof RangeError)} ErrorCtor The error constructor
```

Comment:
Why `typeof`? It should be the `Error` or `RangeError` constructor.

Reply:
from VS code (TypeScript) perspective it looks at `@param {Error}` to be a instance of a Error class, when using typeof then it knows that it can be a Class